pkg/scannerv4/mappers/mappers.go

```diff
@@ -817,8 +817,12 @@ func convertMapToSlice[IN any, OUT any](convF func(*IN) *OUT, in map[string]*IN)
 func fixedInVersion(v *claircore.Vulnerability) string {
 	fixedIn := v.FixedInVersion
 	// Try to parse url encoded params; if expected values are not found leave it.
-	if q, err := url.ParseQuery(fixedIn); err == nil && q.Has("fixed") {
+	q, err := url.ParseQuery(fixedIn)
+	if err == nil && q.Has("fixed") {
 		fixedIn = q.Get("fixed")
+	} else if err == nil && q.Has("lastAffected") {
+		// lastAffected doesn't give us anything informative so return empty string
+		return ""
```

Comment:
probably unlikely to happen, but I wonder if we'd ever see `introduced`, only. In that case, then we will still be showing `introduced=<whatever>`. @daynewlee can we consider a followup to maybe also consider looking for `introduced`?

another consideration:

```
q, err := url.ParseQuery(fixedIn)
if err != nil {
	// v.FixedInVersion is not url encoded, so just return it as-is.
	return fixedIn
}
if q.Has("fixed") {
	return q.Get("fixed")
}
return ""
```